processors/config/rawSvtHits_cfg.py

```diff
@@ -4,7 +4,6 @@
 # Use the input file to set the output file name
 lcio_file = sys.argv[1].strip()
```

Comment:
While you are at it, please update this config file also to use -i and -o options.